Maintainers.txt

```diff
@@ -598,6 +598,10 @@ SecurityPkg: Tcg related modules
 F: SecurityPkg/Tcg/
 R: Rahul Kumar <rahul1.kumar@intel.com> [rahul1-kumar]
 
+SecurityPkg: SVSM related modules
+F: SecurityPkg/Library/Tpm2DeviceLibDTpm/*Svsm*
+R: Oliver Steffen <osteffen@redhat.com> [osteffenrh]
+

```

Comment:
@tlendacky do you want to be listed here too?

Or anybody else?

Reply:
Yes, should probably list me as well.